geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapreduce/GeoMesaInputFormat.scala

```diff
@@ -61,17 +62,32 @@ object GeoMesaInputFormat extends LazyLogging {
    */
   def configure(job: Job, dsParams: Map[String, String], query: Query): Unit = {
 
-    val ds = DataStoreFinder.getDataStore(dsParams).asInstanceOf[AccumuloDataStore]
+    val user = AccumuloDataStoreParams.userParam.lookUp(dsParams).asInstanceOf[String]
+    val password = AccumuloDataStoreParams.passwordParam.lookUp(dsParams).asInstanceOf[String]
+    val isMock = AccumuloDataStoreParams.mockParam.lookUp(dsParams).asInstanceOf[String].toBoolean
+
+    val params: Map[String, Any] =
+      if(isMock) {
+        val connector = new MockInstance("fake").getConnector(user, new PasswordToken(password))
+        connector.securityOperations().changeUserAuthorizations(user, new Authorizations("admin", "user"))
+
+        Map(
+          "connector" -> connector,
+          "caching"   -> false,
+          "tableName" -> AccumuloDataStoreParams.tableNameParam.lookUp(dsParams).asInstanceOf[String]
+        )
```

Comment:
I don't think you need to make these changes either - just checking for the isMock parameter and calling the appropriate `AbstractInputFormat`/`InputFormatBaseAdapter` method should be enough...
if you keep the data store parameters the same as get passed in, it should sync up with the InputFormat class below and the AccumuloDataStoreFactory method